build/Makefile.global

```diff
@@ -142,6 +142,12 @@ prof-clean:
 prof-use:
 	CCACHE_DISABLE=1 $(MAKE) PROF_FLAGS=-fprofile-use all
 
+%_arginfo.h: %.stub.php
```

Comment:
To my knowledge, that would be the first pattern rules, and these are not generally supported; could there be issues with make implementations which don't support them (syntax error or such)?

Reply:
This was my hack to get it working, maybe @petk can suggest how to do this better. I don't know how widely supported pattern rules are.

Also the `touch` below is not great. The intention there is to force a rebuild of the C file if the arginfo file changes -- is there a better way to do that than a (spurious) touch?

Reply:
I think the test here is very simple - basically if it works on FreeBSD with some of its make derivative  and Solaris' default make it works everywhere else.

The pattern rules should be ok...